pkgs/data/fonts/font-awesome/default.nix

```diff
@@ -42,4 +43,9 @@ in {
     version = "5.15.3";
     sha256 = "sha256-EDxk/yO3nMmtM/ytrAEgPYSBbep3rA3NrKkiqf3OsU0=";
   };
+  v6 = font-awesome {
+    version = "6.1.1";
+    sha256 = "sha256-BjK1PJQFWtKDvfQ2Vh7BoOPqYucyvOG+2Pu/Kh+JpAA";
+    rev = "28e297f07af26f148c15e6cbbd12cea3027371d3";
```

Comment:
Since upstream tags release maybe we don't need to manual specify this? :thinking: 

```suggestion
```

Reply:
I guess having https://github.com/FortAwesome/Font-Awesome/releases/tag/6.1.1 is enough :thinking:

Reply:
Nice, didn't seem that! I will update